src/main/resources/db/migration/V005__WorkTable.sql

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Is this for preview ? Will flyway work with two scripts creating the same table ?

Reply:
yeah preview only - its the same table in two different dbs